### PR TITLE
add proc_net_rpc_nfs and nfsd charts info

### PR DESCRIPTION
**Pull request by ilyam8**
##### Summary

This PR adds dashboard info for [proc_net_rpc_nfs](https://github.com/netdata/netdata/tree/master/collectors/proc.plugin/proc_net_rpc_nfs.c) and [nfsd](https://github.com/netdata/netdata/tree/master/collectors/proc.plugin/proc_net_rpc_nfsd.c) collector's charts.

##### Component Name

`web/`

##### Test Plan

Install this PR; check the proc_net_rpc_nfs and nfsd  charts; ensure the info is on the dashboard.

##### Additional Information

Fixes part of netdata/product#2104

### CODE REVIEW

File: web/gui/dashboard_info.js [hunk 1]
```diff
@@ -2789,6 +2789,104 @@ netdataDashboard.context = {
...
+    },
+
+    'nfs.proc3': {
+        info: 'NFSv3 RPC calls. The individual metrics are described in '+
+        '<a href="https://datatracker.ietf.org/doc/html/rfc1813#section-3" target="_blank">RFC1813</a>.'
+    },
+
+    'nfs.proc4': {
+        info: 'NFSv4 RPC calls. The individual metrics are described in '+
+        '<a href="https://datatracker.ietf.org/doc/html/rfc1813#section-3" target="_blank">RFC1813</a>.'
```
Comment (vlvkobal) [inline web/gui/dashboard_info.js:2818]: To provide more information about NFSv4 operations [rfc7530](https://datatracker.ietf.org/doc/html/rfc7530#section-16) (NFSv4.0) and [rfc7862](https://datatracker.ietf.org/doc/html/rfc7862/#section-15) (NFSv4.2) should be added as well, but [rfc8881](https://datatracker.ietf.org/doc/html/rfc8881#section-18) (NFSv4.1) covers most of the operations.
```suggestion
        '<a href="https://datatracker.ietf.org/doc/html/rfc8881#section-18" target="_blank">RFC8881</a>.'
```

File: web/gui/dashboard_info.js [hunk 2]
```diff
@@ -2789,6 +2789,104 @@ netdataDashboard.context = {
...
+    },
+
+    'nfsd.proc3': {
+        info: 'NFSv3 RPC calls. The individual metrics are described in '+
+        '<a href="https://datatracker.ietf.org/doc/html/rfc1813#section-3" target="_blank">RFC1813</a>.'
+    },
+
+    'nfsd.proc4': {
+        info: 'NFSv4 RPC calls. The individual metrics are described in '+
+        '<a href="https://datatracker.ietf.org/doc/html/rfc1813#section-3" target="_blank">RFC1813</a>.'
```
Comment (vlvkobal) [inline web/gui/dashboard_info.js:2882]:
```suggestion
        '<a href="https://datatracker.ietf.org/doc/html/rfc8881#section-18" target="_blank">RFC8881</a>.'
```

File: web/gui/dashboard_info.js [hunk 3]
```diff
@@ -2789,6 +2789,104 @@ netdataDashboard.context = {
...
+    },
+
+    'nfsd.proc4ops': {
+        info: 'NFSv4 RPC operations. The individual metrics are described in '+
+        '<a href="https://datatracker.ietf.org/doc/html/rfc5661#section-18" target="_blank">RFC5661</a>.'
```
Comment (vlvkobal) [inline web/gui/dashboard_info.js:2887]:
```suggestion
        '<a href="https://datatracker.ietf.org/doc/html/rfc8881#section-18" target="_blank">RFC8881</a>.'
```

**Review by vlvkobal — APPROVED** [review]

**Comment by vlvkobal** [comment]
I would also add links for [NFS](https://en.wikipedia.org/wiki/Network_File_System) in the `NFS Server` and `NFS Client` sections.

File: web/gui/dashboard_info.js [hunk 4]
```diff
@@ -2789,6 +2795,104 @@ netdataDashboard.context = {
...
+    },
+
+    'nfs.proc3': {
+        info: 'NFSv3 RPC calls. The individual metrics are described in '+
+        '<a href="https://datatracker.ietf.org/doc/html/rfc1813#section-3" target="_blank">RFC1813</a>.'
+    },
+
+    'nfs.proc4': {
+        info: 'NFSv4 RPC calls. The individual metrics are described in '+
+        '<a href="https://datatracker.ietf.org/doc/html/rfc3010#section-14" target="_blank">RFC3010</a>.'
```
Comment (vlvkobal) [inline web/gui/dashboard_info.js]: rfc3010 is obsoleted by rfc3050 which is obsoleted by rfc7530 and does not include quite a lot of v4.1 operations from rfc8881.

**Review by vlvkobal — APPROVED** [review]

**Review by thiagoftsm — APPROVED** [review]

**Pull request merged**